components/CustomError.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Paper, Alert, Button } from '@mui/material'
+
+const CustomError = ({ title, message, buttonMessage }) => {
+  return (
+    <div style={{ textAlign: 'center' }}>
+      <Paper elevation={ 4 } sx={{ p: 1, backgroundColor: '#FFD700' }}>
+        <h1>{ title }</h1>
+      </Paper>
+      <br />
+      <Paper elevation={ 4 }>
+        <Alert severity="error">{ message }</Alert>
+      </Paper>
+      <br />
+      <Button variant="contained" sx={{ textTransform: 'none' }}>
```

Comment:
why do we need a button with a custom message here?
Also let's add a button allowing us to go back on the previous page if we encounter an error

Reply:
"Why do we need a button with a custom message here?" ->
First of all, I thought that this component can be accessible in several parts of the program. 
So the user will know to which page will be redirected by pressing that button.